frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts

```diff
@@ -4,10 +4,14 @@ import {StateService} from "@uirouter/core";
 import {WorkPackageResource} from "core-app/modules/hal/resources/work-package-resource";
 import {PathHelperService} from "core-app/modules/common/path-helper/path-helper.service";
 import { NgxGalleryOptions, NgxGalleryImage, NgxGalleryAnimation } from 'ngx-gallery';
+import {HttpClient, HttpHeaders} from "@angular/common/http";
+import {CurrentProjectService} from "core-components/projects/current-project.service";
+import {BcfPathHelperService} from "core-app/modules/bcf/helper/bcf-path-helper.service";
+import {RevitBridgeService} from "core-app/modules/bcf/services/revit-bridge.service";
 
 
-export type ViewPointOriginal = { id:string, file_name:string };
-export type ViewPoint = { id:string, fileName:string, fullPath:string };
+export type ViewPointOriginal = { uuid:string, snapshot_id:string, snapshot_file_name:string };
+export type ViewPoint = { snapshotId:string, snapshotFileName:string, snapshotFullPath:string };
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/98c55f584284951e8068eb3ae591271dd16b20b7/frontend/src/app/modules/bcf/bcf-wp-single-view/bcf-wp-single-view.component.ts#L15"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>